app/_includes/md/enterprise/deploy-license.md

```diff
@@ -81,6 +81,8 @@ Result:
 For more detail and options, see the
 [Admin API `licenses` endpoint reference](/gateway/latest/licenses/examples).
 
+It is recommended to [restart](https://docs.konghq.com/gateway/latest/production/running-kong/systemd/#restart-kong-gateway) the Kong Gateway nodes after applying or updating a license.
```

Comment:
```suggestion
We recommend [restarting](/gateway/{{page.release}}/reference/cli/#kong-restart) the {{site.base_gateway}} nodes after applying or updating a license.
```
This might be a better link. The systemd page is a bit too specific.